ocs_ci/deployment/deployment.py

```diff
@@ -285,9 +277,6 @@ def deploy_ocs_via_operator(self):
             }
 
         if self.platform.lower() == constants.VSPHERE_PLATFORM:
-            cluster_data['spec']['monPVCTemplate']['spec'][
```

Comment:
This need to be verified on VmWare if it will be able to get default value for thin.
As I deleted monPVCTemplate from storage cluster CR cause of  it's not defined even when deploying via UI which I mentioned here:  https://bugzilla.redhat.com/show_bug.cgi?id=1770290 . Will verify on VmWare on Monday.